q2_sample_classifier/tests/test_classifier.py

```diff
@@ -870,6 +870,44 @@ def test_predict_feature_order_aint_no_thing(self):
             mse, seeded_predict_results['RandomForestRegressor'])
 
 
+class TestHeatmap(SampleClassifierTestPluginBase):
+
+    def setUp(self):
+        super().setUp()
+        md_ecam = self.get_data_path('ecam_map_maturity.txt')
+        md_ecam = qiime2.Metadata.load(md_ecam)
+        self.md_ecam = md_ecam.get_column('delivery')
+        table_ecam = self.get_data_path('ecam-table-maturity.qza')
+        table_ecam = qiime2.Artifact.load(table_ecam)
+        self.table_ecam, = feature_table.actions.filter_samples(
+            table_ecam, metadata=md_ecam)
+        imp = pd.DataFrame.from_csv(
+            self.get_data_path('importance.tsv'), sep='\t')
+        self.imp = qiime2.Artifact.import_data('FeatureData[Importance]', imp)
+
+    def test_heatmap_default(self):
+        heatmap, table, = sample_classifier.actions.heatmap(
+            self.table_ecam, self.imp, self.md_ecam, group_samples=True)
+        self.assertEqual(table.view(biom.Table).shape, (1056, 2))
+
+    def test_heatmap_importance_threshold(self):
+        heatmap, table, = sample_classifier.actions.heatmap(
+            self.table_ecam, self.imp, self.md_ecam,
+            importance_threshold=0.017)
+        self.assertEqual(table.view(biom.Table).shape, (10, 121))
+
+    # def test_heatmap_feature_count(self):
```

Comment:
Is this supposed to be commented out? Same with L905.

Reply:
crap... thanks for catching (those were commented out while I was building tests). Have fixed in latest commit.